packages/portal/src/components/item/ItemRemoveButton.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <b-button
+      v-b-tooltip.bottom
+      class="remove-button text-uppercase d-inline-flex align-items-center"
+      :class="{ 'button-icon-only': !buttonText }"
+      :variant="buttonVariant"
+      data-qa="item remove button"
+      :aria-label="$t('actions.remove')"
+      :title="$t('account.tooltip.remove')"
+      @click="removeItem"
+    >
+      <span class="icon-remove-circle-outlined" />
+      {{ buttonText ? $t('actions.remove') : '' }}
+    </b-button>
+  </div>
+</template>
+
+<script>
+  import makeToastMixin from '@/mixins/makeToast';
+  import { langMapValueForLocale } from '@europeana/i18n';
+
+  export default {
+    name: 'ItemRemoveButton',
+
+    mixins: [
+      makeToastMixin
+    ],
+
+    props: {
+      /**
+       * Identifier of the item
+       */
+      identifier: {
+        type: String,
+        required: true
+      },
+      /**
+       * Button variant to use for styling the buttons
+       */
+      buttonVariant: {
```

Comment:
I guess this is okay to leave here for consistency with the other user buttons.
Although, since we don't have the list view available on the set pages this won't ever be used.

Reply:
It is used to pass down the correct button variant and keep it aligned with the other buttons. We could hard code it, but like this is a bit more robust for future changes.